code/modules/emoji/emoji_parse.dm

```diff
@@ -19,7 +19,7 @@ var/list/emojis
 			if(search)
 				emoji = lowertext(copytext(text, pos+1, search))
 				if(emoji in emojis)
-					parsed += " <img class=icon src=\ref['icons/emoji.dmi'] iconstate='[emoji]'>"
+					parsed += bicon(icon('icons/emoji.dmi', emoji))
```

Comment:
last time i checked ( it was two weeks ago ) there was.
:supermatter: and :forcesword:
It also breaks door icons on examine on our build.